feed/feed_test.go

```diff
@@ -0,0 +1,138 @@
+package feed
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	indigoTest "github.com/bluesky-social/indigo/testing"
+	"github.com/stretchr/testify/require"
+	"github.com/strideynet/bsky-furry-feed/integration"
+	bffv1pb "github.com/strideynet/bsky-furry-feed/proto/bff/v1"
+	"github.com/strideynet/bsky-furry-feed/store"
+)
+
+const createPostQuery = `
+INSERT INTO
+    candidate_posts (uri, actor_did, created_at, indexed_at, hashtags, tags, has_media, raw)
+VALUES
+    ($1, $2, $3, $4, $5, $6, $7, $8);
+`
+
+func TestChronologicalGenerator(t *testing.T) {
+	if testing.Short() {
```

Comment:
FYI we need to change the go test CI job as it currently runs `-short` for tests outside of `integration/`. I think at our current stage, we can just make one test job that runs all tests rather than sticking to the two seperate test jobs we have atm.